include/openssl/ct.h

```diff
@@ -398,7 +402,7 @@ int SCT_LIST_validate(const STACK_OF(SCT) *scts, CT_POLICY_EVAL_CTX *ctx);
  * Returns < 0 on error, >= 0 indicating bytes written (or would have been)
  * on success.
  */
-int i2o_SCT_LIST(const STACK_OF(SCT) *a, unsigned char **pp);
+__owur int i2o_SCT_LIST(const STACK_OF(SCT) *a, unsigned char **pp);
```

Comment:
Should I also rename {i2o,o2i}_SCT? This pair of functions are mostly wrappers around them, that just deal with length of the SCT list at the start, so it'd be odd for them to have wildly different names. The "octet string" in this case is actually TLS wire format - I expect it's safe to say there are other functions in OpenSSL doing TLS encoding, so is there an existing naming convention? Two obvious options are either the cryptic "i2t", which is at least consistent with the prevailing naming scheme for encoding/decoding functions, or something like "SCT_LIST_to_tls", which is a bit more self-explanatory.

I assume the "i2d_SCT_LIST" and related functions should remain named as they are, for consistency with the other internal->DER encoding functions?